google/cloud/bigquery/table.py

```diff
@@ -291,16 +284,36 @@ class Table(object):
     """
 
     _PROPERTY_TO_API_FIELD = {
+        "clustering_fields": "clustering",
+        "created": "creationTime",
+        "dataset_id": ["tableReference", "datasetId"],
+        "description": "description",
         "encryption_configuration": "encryptionConfiguration",
+        "etag": "etag",
         "expires": "expirationTime",
         "external_data_configuration": "externalDataConfiguration",
         "friendly_name": "friendlyName",
+        "full_table_id": "id",
+        "labels": "labels",
+        "location": "location",
+        "modified": "lastModifiedTime",
         "mview_enable_refresh": "materializedView",
+        "mview_last_refresh_time": ["materializedView", "lastRefreshTime"],
         "mview_query": "materializedView",
         "mview_refresh_interval": "materializedView",
+        "num_bytes": "numBytes",
+        "num_rows": "numRows",
         "partition_expiration": "timePartitioning",
         "partitioning_type": "timePartitioning",
+        "project": ["tableReference", "projectId"],
+        "range_partitioning": "rangePartitioning",
+        "time_partitioning": "timePartitioning",
+        "schema": "schema",
+        "streaming_buffer": "streamingBuffer",
+        "self_link": "selfLink",
+        "table_id": ["tableReference", "tableId"],
         "time_partitioning": "timePartitioning",
+        "type": "type",
```

Comment:
I'm pretty sure we fallback to the passed-in name if we don't find it here, but I'm fine including everything for consistency.

Reply:
... and _explicit is better than implicit_. :)

(also, the [routine](https://github.com/googleapis/python-bigquery/blob/0abb56669c097c59fbffce007c702e7a55f2d9c1/google/cloud/bigquery/routine/routine.py#L41-L54) implementation takes the same approach)